siluria.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+title: Planet Siluria
+---
+
+# Planet Siluria
+
+- This world is limited to Jungles.
+- Structures will generate automatically as you explore the world.
+- Mobs will spawn in these structures — you will need to have TARDISWeepingAngels plugin installed and the TARDISWeepingAngels-Resource-Pack to see them properly.
+- A random loot chest is also located in the structures.
+
+## Enabling Siluria
+
+Siluria is disabled by default, here are some instructions on enabling it.
+
+### Minimum requirements
```

Comment:
Think we could probably remove this section (on all pages)

Reply:
Do you think we should have a custom planets page that lists all 3, then the config and stuff at the bottom (since it's generic except for Skaro acid)

Reply:
Happy for you do whatever you think best, but sounds like a good plan to me.